ServiceInnovationLab/pancake-backend
# API for rebate form

# What has changed and why?
Save a rebate form (no signatures yet)
Retrieve the same form by token 

# How to test this change
Example request here 
https://documenter.getpostman.com/view/3560032/pancake/RVnbBHcz#3d9ada3f-cf96-ea3f-deb4-928c2ccfc8a4



* closes #2  api method to call to save an application
* closes #3 As an applicant, I receive a token (e.g. ABC) and take this with me to the city council office, so the CC staff can use token to look up the data I've entered and present it ready to be signed.
* closes #5 as an applicant, (or staff member helping), I can enter a "token" into the ipad/tablet at the CC service centre, and the application form is displayed ready to be signed by me and my witness


Example payload to create a form
https://pancakenz-staging.herokuapp.com/api/docs#/endpoints/-v1-rebate_forms-post
```
{
  "data": {
    "type": "rebate_forms",
    "attributes": {
      "valuation_id": "abc",
       "foo": "bar"
    }
  }
}
```
